models/discord_test.go

```
@@ -41,4 +42,35 @@ func TestSessionWrapper(t *testing.T) {
 		}, "should panic when getUerId() is called")
 	})
 
+	t.Run("SessionWrapper should always implement guildMemberNickname() method", func(t *testing.T) {
+		assert.Panics(t, func() {
+			sessionWrapper.GuildMemberNickname("1", "2")
```

golangci-lint (1.64.8)

Error return value of `sessionWrapper.GuildMemberNickname` is not checked

(errcheck)